api/tests/v2/test_web_tokens.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import resolve
+from rest_framework.test import APITestCase
+
+
+class WebTokenTests(APITestCase):
+    def test_url_resolution(self):
+        """
+        Test that the url config correctly resolves uuids
+        """
+        match = resolve('/api/v2/web_tokens/d71e6b24-841e-4b77-a819-01098579e6a9')
```

Comment:
Could you test what happens with URLs which have query string parameters?

Reply:
Good point